### DISCUSSION TITLE
Play Count

### DISCUSSION THREAD

**Discussion by iamtheplauge** [watsonbox/exportify#99] [Ideas]
With the most recent Spotify update for desktop, the play count is now visible for all songs (without have to hover over the old popularity bar). I am curious as to whether it is now possible to export this piece of metadata with a song, or whether it is still inaccessible. If it is possible to add the play count, I would absolutely love that to be included.

Thanks again for this amazing tool. I use it a bunch!

**Comment by watsonbox** [comment]
It does seem to be a popular request but sadly I don't believe it's available in the API 😞 https://github.com/spotify/web-api/issues/70

**Reply by iamtheplauge** [reply]
Oof, that's such a shame. One day maybe...
